src/game-of-life/setup.md

```diff
@@ -116,10 +116,13 @@ greet("Rust and WebAssembly");
 
 ### `src/lib.rs`
 
-The `src/lib.rs` file is the root of the Rust crate that we are compiling to
-WebAssembly. It uses `wasm_bindgen` to interface with JavaScript. It imports the
-`window.alert` JavaScript function, and exports the `greet` Rust function, which
-takes a `name` parameter and alerts a greeting message.
+We will use [wasm-bindgen](https://github.com/rustwasm/wasm-bindgen) to
+communicate between Rust/wasm and JavaScript. `wasm-bindgen` currently relies
```

Comment:
`WASM` ?

Reply:
Hey @sendilkumarn. Sorry, I don't understand your comment. Is there an issue with the above?

Reply:
Oops sorry, I should have been clear. It should be `WASM` instead of `wasm` right?

Reply:
Grepping the code base, it seems that "wasm" is the preferred choice rather than "WASM".